packages/now-static-build/src/frameworks.ts

```diff
@@ -42,26 +40,33 @@ const frameworkList: Framework[] = [
     dependency: 'gatsby',
     buildCommand: 'gatsby build',
     getOutputDirName: async () => 'public',
-    defaultRoutes: async (dirPrefix: string) => {
-      try {
-        const nowRoutesPath = join(
-          dirPrefix,
-          'public',
-          '__now_routes_g4t5bY.json'
```

Comment:
I think we might need to keep this for backwards compatibility.

@lucleray What do you think?

Reply:
It's not used by a lot of users, but I'd still keep it 👍

Reply:
It's used in:
https://www.npmjs.com/package/gatsby-plugin-now
https://www.npmjs.com/package/gatsby-plugin-zeit-now